src/main/paradox/docs/api/iam/iam-acls-api.md

```diff
@@ -1 +1,206 @@
-# Access Control List
\ No newline at end of file
+# Access Control Lists
+
+Access Control Lists are rooted in the `/v1/acls` collection.
+
+An ACL defines the applications' data access restriction using the following three parameters:
+         
+- permission: the value used to limit application's access to information.
```

Comment:
I would say: 

- permission: the value used to limit a client (user, group) access to resources.